### ISSUE TITLE
why multiplier must be greater than 1.0?

### ISSUE THREAD

**Issue by insikk** [ildoonet/pytorch-gradual-warmup-lr#4]
I want to use this linear warmup for gradually increasing learning rate from 0 to base_lr, which involves multiplier being 1.0. 

However, this code enforce us using multiplier greater than 1.0. 
Do we really need this restriction?

**Comment by ildoonet** [comment]
Hello, this code increases the learning rate from 0 to base_lr * multiplier. This is because the original paper was using this method to train with higher learning rate according to multiplied batch size.

But your request seems reasonable, I will fix the codes to accept multiplier=1.0.
